### DISCUSSION TITLE
questions about `Terminal Character`

### DISCUSSION THREAD

**Discussion by eatgrass** [zellij-org/zellij#2724] [General]
I am attempting to utilize Zellij as a replacement for my current Tmux setup, and I have a few inquiries regarding character rendering:

It seems that Zellij is unable to display characters with the undercurl and underdotted attributes correctly.

Additionally, it appears that Zellij struggles to accurately color the underlined text in accordance with Neovim's sp highlight argument.

In my opinion, a more effective approach to character rendering would be to delegate this task to the terminal emulator's $TERM itself. Changing the rendering style would then involve reading the terminfo database to output ANSI escape codes accordingly.

**Comment by morenohernan** [comment]
Never mind, found it.. how to do it with a layout..

**Reply by eatgrass** [reply]
I believe that dealing with escape codes is not appropriate for a terminal multiplexer; this seems more like something a terminal emulator should handle. Furthermore, I don't think it's closely related to layout either, just think about TMUX

**Reply by zicklag** [reply]
@morenohernan do you have a config sample of getting the underlining to work?
